extensions/resource-deployment/src/ui/deployClusterWizard/pages/summaryPage.ts

```diff
@@ -342,6 +342,7 @@ export class SummaryPage extends WizardPageBase<DeployClusterWizard> {
 		};
 		const storageTable = this.view.modelBuilder.table().withProperties<azdata.TableComponentProperties>({
 			title: localize('deployCluster.StorageSettings', "Storage settings"),
+			ariaLabel: localize('deployCluster.StorageSettings', "Storage settings"),
```

Comment:
Just make a single instance of the string that both this and the title use. I'd also suggest moving it out into https://github.com/microsoft/azuredatastudio/blob/master/extensions/resource-deployment/src/localizedConstants.ts - this makes it easier to find duplicates and makes it so we don't have to do the lookup multiple times.